documentation/modules/oauth/proc-oauth-authorization-broker-config.adoc

```diff
@@ -83,6 +84,7 @@ Default is `false`.
 <9> (Optional) The number of threads to use to refresh (in parallel) the grants for the active sessions. The default value is 5.
 <10> (Optional) The connect timeout in seconds when connecting to the Keycloak token endpoint. The default value is 60.
 <11> (Optional) The read timeout in seconds when connecting to the Keycloak token endpoint. The default value is 60.
+<12> (Optional) The maximum number of retries to perform (without pausing) if the initial session grants request to the authorization server fails in unexpected way. The default value is 0 (no retries). When using this option consider reducing the timeout times for `connectTimeoutSeconds` and `readTimeoutSeconds`. Note that retries prevent the current worker thread from being available to other requests. Many of them stalling could make the Kafka broker unresponsive.
```

Comment:
```suggestion
<12> (Optional) The maximum number of times to retry (without pausing) a failed HTTP request to the authorization server. The default value is `0`, meaning that no retries are performed. To use this option effectively, consider reducing the timeout times for the `connectTimeoutSeconds` and `readTimeoutSeconds` options. However, note that retries may prevent the current worker thread from being available to other requests, and if too many requests stall, it could make the Kafka broker unresponsive.
```